include/deal.II/lac/sparse_vanka.h

```diff
@@ -376,9 +384,9 @@ class SparseVanka
  * Splitting the matrix into blocks is always done in a way such that the
  * blocks are not necessarily of equal size, but such that the number of
  * selected degrees of freedom for which a local system is to be solved is
- * equal between blocks. The reason for this strategy to subdivision is
- * load-balancing for multithreading. There are several possibilities to
- * actually split the matrix into blocks, which are selected by the flag @p
+ * equal between blocks. The reason for this strategy to subdivision is load-
+ * balancing for multithreading. There are several possibilities to actually
```

Comment:
This will lead to a space between "load-" and "balancing". Please leave the word on the next line.